sdk/ml/azure-ai-ml/azure/ai/ml/constants/_registry.py

```diff
@@ -5,6 +5,13 @@
 from enum import Enum
 
 
+class ManagedServiceIdentityType(str, Enum):
+    """Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed)."""
+
+    NONE = "None"
```

Comment:
Don't you need userassigned?

Reply:
Mention of user assigned type was accidentally left in from Workspace logic. Removed it to avoid confusion. Registries always expects system assigned identity.